examples/pyteal/scripts/py-teal.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Description:
+ * This file demonstrates the PyTeal Example for HLTC(Hash Time Lock Contract)
+*/
+const { transferMicroAlgos, transferAsset, balanceOf, ASC1Mode } = require("algob");
+
+async function run(runtimeEnv, deployer) {
+
+  const masterAccount = deployer.accountsByName.get("master-account")
+  const bobAccount = deployer.accountsByName.get("bob-account"); // Buyer
+
+  await transferMicroAlgos(deployer, masterAccount, bobAccount.addr, 200000000, {note: "funding account"});  
+
+  // Secret value // Image : QzYhq9JlYbn2QdOMrhyxVlNtNjeyvyJc/I8d8VAGfGc=
```

Comment:
is the `Qz...` a base64 encoding?

Reply:
secret value hashed with sha256 will produce our image hash. which will be used in Contract code for later matching

Reply:
adding a more elaborated comment